### PR TITLE
Adding rk3588-firmware

### PR DESCRIPTION
**Pull request by stephenquoll**

### CODE REVIEW

File: pkgs/rk3588-firmware/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{ pkgs, stdenv, ... }:
```
Comment (SomeoneWeird) [inline pkgs/rk3588-firmware/default.nix:1]: maybe set these with the defaults like the rest of the packages

File: pkgs/rk3588-firmware/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+stdenv.mkDerivation rec {
+  pname = "rk3588-firmware";
+  version = "2024.09.14";
+
+  src = fetchGit {
+    url = "https://github.com/friendlyarm/sd-fuse_rk3588.git";
```
Comment (SomeoneWeird) [inline pkgs/rk3588-firmware/default.nix:8]: does this need to be your fork? we should probably move it to the org if so

Comment (stephenquoll) [reply pkgs/rk3588-firmware/default.nix:8]: This is the official source (it uses the firmware unmodified).  I'm expecting my fork to be replaced by extra code in the dogebox repo.

File: pkgs/rk3588-firmware/default.nix (hunk 1 above)
Comment (stephenquoll) [reply pkgs/rk3588-firmware/default.nix:1]: Will do!

**Pull request merged**